core/domain/html_validation_service.py

```diff
@@ -831,3 +831,69 @@ def get_filename_with_dimensions(old_filename, exp_id):
     new_filename = regenerate_image_filename_using_dimensions(
         old_filename, height, width)
     return new_filename
+
+
+def add_image_id_and_remove_filepath_from_image_tag(
+        html, image_info_dict_1, image_info_dict_2):
```

Comment:
Please make your variable name self explanatory! (I'm not sure what's the difference in the two dict, even docstring isn't clear).

Reply:
done